node/packages/parser/src/ethereum/index.ts

```diff
@@ -114,7 +123,7 @@ export class TransactionParser {
     return transfers
   }
 
-  private getStatus(tx: Tx): Status {
+  private static getStatus(tx: Tx): Status {
```

Comment:
I have noticed a couple updates to static, just curious as to your pattern on making a function static. Anything that doesn't reference this?

I would say that in this case, making a private function static doesn't provide us with any benefit that I know know of.

Reply:
Ah yes, it's a convention I usually follow for methods that don't touch any state of the class instance. The method, therefore, sits on the class, rather than the instance of the class.

In theory, there is a small, but negligible performance improvement, but I just use it as a way to help conceptualise the level at which the method operates.

I'm cool either way, but I _think_ using [static](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static) is more idiomatic TypeScript.

Reply:
That works with me, thanks for the breakdown!